internal/proxy/proxy_test.go

```diff
@@ -77,7 +77,7 @@ func TestProxy(t *testing.T) {
 
 		// the stored request should be the one _from the proxy_ since it's stored by
 		// the (fake) vault.
-		require.NotEmpty(t, vaultRequests[0].Header.Get("User-Agent"))
+		require.Contains(t, vaultRequests[1].Header.Get("User-Agent"), proxyUserAgent)
```

Comment:
This is [1] because another request from a raw agent also shows up (with user agent Go-http-client)